docs/050-breaking-changes.rst

```diff
@@ -327,7 +327,7 @@ New version:
 
 ::
 
-   pragma solidity >0.4.25;
+   pragma solidity >=0.4.26 <0.6.0;
```

Comment:
We actually could use something like `>0.4.99 <0.6.0`.

Reply:
I was going to change the "0.5.0" ones to `>0.4.42 <0.6.0`, added as a distinct commit here.

Reply:
I'm fine with whatever :)